mir_perception/mir_handle_detection/ros/src/drawer_handle_perceiver.cpp

```diff
@@ -214,6 +212,13 @@ void DrawerHandlePerceiver::extractPlaneOutlier(const PCloudT::Ptr &input, PClou
     this->project_inliers.setIndices(inliers);
     this->project_inliers.filter(*pc_plane);
 
+    Eigen::Vector4f normal_vector(0.0, 0.0, 0.0, 0.0);
+    bool cluster_success = this->getDrawerPlaneNormal(pc_plane, normal_vector);
```

Comment:
the normal of the plane is already contained in `coefficients`, so you don't need to calculate it again (see [here](http://docs.pointclouds.org/1.7.0/group__sample__consensus.html))

Reply:
Done